common/app/views/support/HtmlCleaner.scala

```diff
@@ -179,87 +177,6 @@ case class PictureCleaner(article: Article, amp: Boolean)(implicit request: Requ
   }
 }
 
-case class LiveBlogDateFormatter(isLiveBlog: Boolean)(implicit val request: RequestHeader) extends HtmlCleaner  {
-  def clean(body: Document): Document = {
-    if (isLiveBlog) {
-      body.select(".block").foreach { el =>
-        val id = el.id()
-        el.select(".block-time.published-time time").foreach { time =>
-          val datetime = DateTime.parse(time.attr("datetime"))
-          val hhmm = Format(datetime, "HH:mm")
-          time.wrap(s"""<a href="#$id" itemprop="url" class="block-time__link"></a>""")
-          time.attr("data-relativeformat", "med")
-          time.after( s"""<span class="block-time__absolute">$hhmm</span>""")
-          if (datetime.isAfter(DateTime.now().minusDays(5))) {
-            time.addClass("js-timestamp")
-          }
-        }
-      }
-    }
-    body
-  }
-}
-
-case class LiveBlogLinkedData(isLiveBlog: Boolean)(implicit val request: RequestHeader) extends HtmlCleaner  {
-  def clean(body: Document): Document = {
-    if (isLiveBlog) {
-      body.select(".block").foreach { el =>
-        val id = el.id()
-        val blockElements = el.select(".block-elements")
-        val noByline = el.attributes().get("data-block-contributor").isEmpty
-        el.attr("itemprop", "liveBlogUpdate")
-        el.attr("itemscope", "")
-        el.attr("itemtype", "http://schema.org/BlogPosting")
-        el.select(".block-time.published-time time").foreach { time =>
-          time.attr("itemprop", "datePublished")
-        }
-        el.select(".block-title").foreach { title =>
-          title.attr("itemprop", "headline")
-
-        }
-        if (noByline) blockElements.addClass("block-elements--no-byline")
-        blockElements.foreach { body =>
-          body.attr("itemprop", "articleBody")
-        }
-      }
-    }
-    body
-  }
-}
-
-case class BloggerBylineImage(article: Article)(implicit val request: RequestHeader) extends HtmlCleaner  {
-  def clean(body: Document): Document = {
-    if (article.isLiveBlog) {
-      body.select(".block").foreach { el =>
-        val contributorId = el.attributes().get("data-block-contributor")
-        if (contributorId.nonEmpty) {
-          article.tags.tags.find(_.id == contributorId).map{ contributorTag =>
-            val html = views.html.fragments.meta.bylineLiveBlockImage(contributorTag)
-            el.getElementsByClass("block-elements").headOption.foreach(_.before(html.toString()))
-          }
-        }
-      }
-    }
-    body
-  }
-}
-
-case class LiveBlogShareButtons(article: Article)(implicit val request: RequestHeader) extends HtmlCleaner  {
```

Comment:
still seems to work:
![image](https://cloud.githubusercontent.com/assets/7304387/12172822/2c3f3d7e-b54b-11e5-87e1-e7717f1a7c05.png)
it's just done in a play template based on the capi blocks rather than being mixed in to the html in the cleaner